content/blogs/newsletter-102.md

```diff
@@ -0,0 +1,113 @@
+---
+title:       "Newsletter 102 - 2022/2023 news"
+author:      "GeoB99"
+date:        2023-03-25
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers and enthusiasts! It has been quite a long time since we published the 101th newsletter and so far no further updates have been posted since then.
```

Comment:
```suggestion
Hello ReactOS followers and enthusiasts! It has been quite a long time since we published the [101th newsletter](/blogs/newsletter-101/) and so far no further updates have been posted since then.
```